hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/GetPartitionsMessageTask.java

```diff
@@ -61,7 +62,7 @@ protected ClientMessage call() {
             Address owner = partitions[i].getOwnerOrNull();
             int index = -1;
             if (owner == null) {
-                return null;
+                return VoidResultParameters.encode();
```

Comment:
This does not look like it is going to work correctly. We will get exception when trying to read the result here:
https://github.com/hazelcast/hazelcast/blob/master/hazelcast-client-new/src/main/java/com/hazelcast/client/spi/impl/ClientPartitionServiceImpl.java#L103

I solved similar problem in map.getEntryView by adding a boolean to parameters to differentiate null or not.